### ISSUE TITLE
Add logging output to better understand how glass tracks time

### ISSUE THREAD

**Issue by rebelzach** [timeglass/glass#34]
Some simple output to a log file somewhere would be nice so that I can verify I understand how glass is interpreting my behavior.

**Comment by advdv** [comment]
Hey There! This sound like something everyone could benefit from, but could you elaborate a bit on what you expect to see in such log? or maybe an example on what you would like to interpret or gain from reading such log?

**Comment by rebelzach** [comment]
I'm thinking something like this

```
Jun  3 09:30:27 Timer Started : Switched to branch 'fubar-hotfix'
Jun  3 09:35:27 Timer Paused : File system inactive for 5 minutes
Jun  3 09:42:02 Timer Resumed : Activity on file '/fubar.c'
Jun  3 09:42:02 Timer Reset : Received command 'glass lap'
```

**Comment by advdv** [comment]
Sounds good, will schedule this for 0.5.3, the release after the next release (which is focussed on cross platform support -> tough cookie).

**Comment by advdv** [comment]
@rebelzach , this has been implemented in the latest version (0.6.0), the logs can be found in the system wide Timeglass directory: "/Library/Timeglass" on osx. Please try it out and let me know what you think!
